CMakeLists.txt

```diff
@@ -201,6 +211,14 @@ add_dependencies(psi4_core ambit_external
                            libint_external
                            pcmsolver_external
                            pybind11_external)
+if(ENABLE_psi4pasture)
+  ExternalProject_Add(psi4pasture
+    GIT_REPOSITORY https://github.com/amjames2/psi4pasture
```

Comment:
Of the usual ones from, for example, [libefp](https://github.com/psi4/psi4/blob/master/external/libefp/CMakeLists.txt#L16-L31), you'll want to include at least https://github.com/psi4/psi4/blob/master/external/libefp/CMakeLists.txt#L16-L19, L22, L31 & CXX analogs for compiler and flags.